packages/portfolio-contract/test/supports.ts

```diff
@@ -213,13 +190,15 @@ export const setupPortfolioTest = async ({
     bootstrap: { agoricNamesAdmin, bankManager },
   } = common;
   const usdc = withAmountUtils(makeIssuerKit('USDC'));
+  const bld = withAmountUtils(makeIssuerKit('BLD'));
   const poc26 = withAmountUtils(makeIssuerKit('Poc26'));
   await E(bankManager).addAsset(
     uusdcOnAgoric,
     'USDC',
     'USD Circle Stablecoin',
     usdc.issuerKit,
   );
+  await E(bankManager).addAsset('ubld', 'BLD', 'BLD', bld.issuerKit);
   await E(bankManager).addAsset(
     'upoc26',
```

Comment:
No decimals, so poc26 is not a micro-amount.

```suggestion
    'poc26',
```

Reply:
oops.

it's in the devnet vbank as `upoc26`. IOU an issue or something
cc @LuqiPan